### PR TITLE
Update privacy policy

### PR DESCRIPTION
**Pull request by rubenwardy**

### CODE REVIEW

File: app/flatpages/privacy_policy.md [hunk 1]
```diff
@@ -32,29 +33,40 @@ Please avoid giving other personal information as we do not want it.
...
     * Send notifications - the user may configure this to their needs, including opting out.
 * Passwords are used to authenticate the user.
+* The audit log is used to record actions that may be harmful
 * Other information is displayed as part of ContentDB's service.
 
 ## Who has access
 
 * Only the admin has access to the HTTP requests.
   The logs may be shared with others to aid in debugging, but care will be taken to remove any personal information.
+* Encrypted backups are shared with Minetest staff members.
```
Comment (SmallJoker) [inline app/flatpages/privacy_policy.md:43]: Who is `staff`? Where is it documented?

Comment (rubenwardy) [reply app/flatpages/privacy_policy.md:43]: Moderators and core developers. I currently don't do this with anyone, this is to allow it to be done in the future

Comment (rubenwardy) [reply app/flatpages/privacy_policy.md:43]:
```suggestion
* Encrypted backups may be shared with select Minetest staff members (moderators + core devs).
```

File: app/flatpages/privacy_policy.md [hunk 2]
```diff
@@ -32,29 +33,40 @@ Please avoid giving other personal information as we do not want it.
...
     * Send notifications - the user may configure this to their needs, including opting out.
 * Passwords are used to authenticate the user.
+* The audit log is used to record actions that may be harmful
 * Other information is displayed as part of ContentDB's service.
 
 ## Who has access
 
 * Only the admin has access to the HTTP requests.
   The logs may be shared with others to aid in debugging, but care will be taken to remove any personal information.
+* Encrypted backups may be shared with select Minetest staff members (moderators + core devs).
```
Comment (rubenwardy) [inline app/flatpages/privacy_policy.md:43]:
```suggestion
* Encrypted backups may be shared with selected Minetest staff members (moderators + core devs).
```

**Review by SmallJoker — APPROVED** [review]
LGTM

**Pull request merged**